files/packit.wsgi

```diff
@@ -1,3 +1,5 @@
 # -*- coding: utf-8 -*-
+import sys
 
+sys.path.insert(0, "/usr/share/packit")
```

Comment:
why not set PYTHONPATH variable in the deployment?

Reply:
I have to test it and find out the way. Will see.